plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergColumnHandle.java

```diff
@@ -184,4 +186,35 @@ public String toString()
     {
         return getId() + ":" + getName() + ":" + type.getDisplayName();
     }
+
+    public static IcebergColumnHandle pathColumnHandle()
```

Comment:
These could be static constants instead of methods

Reply:
as we discussed, the method looks ok to me, so can we stick to the methods?